fractions/wildfly/elytron/src/main/java/org/wildfly/swarm/elytron/ElytronFraction.java

```diff
@@ -133,40 +135,48 @@ public ElytronFraction applyDefaults() {
             realm.className("org.wildfly.swarm.elytron.runtime.Realm");
```

Comment:
Again, hard to know what's changed with the altering of the call sequence

Reply:
I actually didn't change ordering here (with one little exception), there were just too many differences.